index.html

```diff
@@ -1431,7 +1432,26 @@ <h5>How to's</h5>
  </section>
  </section>
 </section>  
-  
+
+<section id="cursive" class="subtopic">
+  <h3>Cursive text</h3>
+  <p class="advisement" id="cursive_opacity"><a class="self" href="#cursive_opacity">&#x200B;</a>
+    Overlapping joinings should not be exposed when transparency is applied to the cursive text such as Arabic, Mongolian, and N'Ko.
+  </p>
+  <p class="advisement" id="cursive_border"><a class="self" href="#cursive_border">&#x200B;</a>When adding text border, a joined letter should not be separated from its joined neighbors in cursive text.</p>
```

Comment:
how about 
"When adding a text stroke or shadow, joined letters should not be separated from their neighbors in cursive script text."

Reply:
Looks good to me. Updated.